Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/DataEditorForm.java

```diff
@@ -445,8 +445,12 @@ void setProcess(Process process) {
         this.process = process;
     }
 
-    void switchStructure(Object treeNodeData) throws InvalidMetadataValueException, NoSuchMetadataFieldException {
-        metadataPanel.preserveLogical();
+    void switchStructure(Object treeNodeData) throws NoSuchMetadataFieldException {
+        try {
+            metadataPanel.preserveLogical();
+        } catch (InvalidMetadataValueException e) {
+            logger.info(e.getLocalizedMessage());
```

Comment:
Stack trace is nice to have (but not necessary):

`logger.info(e.getLocalizedMessage(), e);`